### DISCUSSION TITLE
usage of Day/night difference

### DISCUSSION THREAD

**Discussion by der-manu** [theyosh/TerrariumPI#678] [Q&A] [answered]
Hi
I have a little confusion with the correct usage.
In my setup there is a DHT22 as sensor for temperature and the alarm min=26; alarm max=28. It Works as expected an keeps the temperaure right. In nighttimes I like to have 4° less (22-24°C).
<img width="1125" alt="image" src="https://user-images.githubusercontent.com/6383246/158648065-1077f00a-fe2b-4a82-9e40-f0fb1692ec38.png">
Wich is the right setting to this? do I set -4° because I want 4°less @night or do I set sensor-setting to night temperature and set 
Day/night difference to +4°?
Another question; is it possible to set Language to english but currency Sign to €? I did not find that setting.

**Comment by der-manu** [comment, marked as answer]
ok, i could have found https://theyosh.github.io/TerrariumPI/setup/#areas and realize that -4 should do the trick.
sorry for bothering

**Reply by theyosh** [reply]
Nice you found it. About the currency, at is not possible to have the combination of English language and euro sign. But I will look into it with the new GUI that I am currently making. Now the currency is bound to the language.
